servo_node:
  ros__parameters:
    velButton: 6
    disButton: 1
    ServoButton: [2,2,2,2,0,0,0,0] #[1番,2番,3番,4番,5番,6番,7番,8番] 0 or 1
    ServoMode: ["Toggle","Toggle","Toggle","Toggle","Toggle","Toggle","Toggle","Toggle"]